src/middlewared/middlewared/plugins/support.py

```diff
@@ -172,10 +169,9 @@ async def fetch_categories(self, token):
         'new_ticket',
         Str('title', required=True, max_length=None),
         Str('body', required=True, max_length=None),
-        Str('category', required=True),
```

Comment:
The enterprise systems still use the `category` field to submit new tickets. It may be better to make it optional.